backend/src/main/java/ai/verta/modeldb/authservice/RoleServiceUtils.java

```diff
@@ -1148,6 +1150,57 @@ public WorkspaceDTO getWorkspaceDTOByWorkspaceId(
     }
   }
 
+  private ResourceVisibility getResourceVisibility(
+      WorkspaceType workspaceType, ProjectVisibility projectVisibility) {
+    if (workspaceType == WorkspaceType.ORGANIZATION) {
+      if (projectVisibility == ProjectVisibility.ORG_SCOPED_PUBLIC) {
+        return ResourceVisibility.ORG_SCOPED_PUBLIC;
+      }
+      return ResourceVisibility.ORG_DEFAULT;
+    }
+    return ResourceVisibility.PRIVATE;
+  }
+
+  @Override
+  public boolean createWorkspaceRoleBinding(
+      Long workspaceId,
+      WorkspaceType workspaceType,
+      String resourceId,
+      Long ownerId,
+      ModelDBServiceResourceTypes resourceType,
+      CollaboratorType collaboratorType,
+      ProjectVisibility visibility) {
+    try (AuthServiceChannel authServiceChannel = new AuthServiceChannel()) {
+      LOGGER.info("Calling CollaboratorService to create resources");
+      ResourceType modeldbServiceResourceType =
+          ResourceType.newBuilder().setModeldbServiceResourceType(resourceType).build();
+      Resources resources =
+          Resources.newBuilder()
+              .setResourceType(modeldbServiceResourceType)
+              .setService(Service.MODELDB_SERVICE)
+              .addResourceIds(resourceId)
+              .build();
+      ResourceVisibility resourceVisibility = getResourceVisibility(workspaceType, visibility);
+      SetResources setResources =
+          SetResources.newBuilder()
+              .setWorkspaceId(workspaceId)
+              .setOwnerId(ownerId)
```

Comment:
I believe that UAC will auto-populate the owner, so we don't _need_ to send it here. No harm in sending as long as it's correct though.

Reply:
I made the owner ID optional